### DISCUSSION TITLE
Ability to use Hotkeys for Multiple Programs Simultaneously

### DISCUSSION THREAD

**Discussion by FroobTubeLIVE** [radj307/volume-control#53] [Ideas & Feature Discussions]
If you were able to make it so that multiple programs can be controlled at the same time, instead of having to cycle through "sessions", this would be a one-stop solution for so many people.

Much love for the work you've done so far 💚 and happy holidays :)

**Comment by StormingChaos** [comment]
This is now issue #54 

Thanks for the suggestion!
